include/autocluster.hrl

```diff
@@ -35,6 +35,7 @@
          {config, consul_svc_addr,       "CONSUL_SVC_ADDR",        "undefined",  string,  false},
          {config, consul_svc_addr_auto,  "CONSUL_SVC_ADDR_AUTO",   false,        atom,    false},
          {config, consul_svc_addr_nic,   "CONSUL_SVC_ADDR_NIC",    "undefined",  string,  false},
+         {config, consul_svc_addr_nodename,   "CONSUL_SVC_ADDR_NODENAME",    false,  atom,  false},
```

Comment:
Since we have `USE_LONGNAME` can we use directly it without add another variable ? `CONSUL_SVC_ADDR_NODENAME`, WDYT?